lmdeploy/model.py

```diff
@@ -492,6 +492,44 @@ def match(cls, model_path: str) -> Optional[str]:
             return 'llama2'
 
 
+@MODELS.register_module(name='llama3')
+class Llama3(BaseChatTemplate):
+    """Chat template of LLaMA3 model."""
+
+    def __init__(self,
+                 system='<|start_header_id|>system<|end_header_id|>\n\n',
+                 meta_instruction=None,
+                 eosys='<|eot_id|>',
+                 assistant='<|start_header_id|>assistant<|end_header_id|>\n\n',
+                 eoa='<|eot_id|>',
+                 user='<|start_header_id|>user<|end_header_id|>\n\n',
+                 eoh='<|eot_id|>',
+                 stop_words=['<|eot_id|>', '<|end_of_text|>'],
+                 session_len=8192,
+                 **kwargs):
+        super().__init__(system=system,
+                         meta_instruction=meta_instruction,
+                         eosys=eosys,
+                         assistant=assistant,
+                         eoa=eoa,
+                         user=user,
+                         eoh=eoh,
+                         stop_words=stop_words,
+                         session_len=session_len,
+                         **kwargs)
+
+    @classmethod
+    def match(cls, model_path: str) -> Optional[str]:
+        """Return the model_name that was registered to MODELS.
+
+        Args:
+            model_path (str): the model path used for matching.
+        """
+        if 'llama-3' in model_path.lower() or 'llama3' in model_path.lower():
+            if 'instruct' in model_path.lower():
```

Comment:
Why increase the condition on `instruct`?

Reply:
Have you tested all four of them?
```
meta-llama/Meta-Llama-3-8B
meta-llama/Meta-Llama-3-8B-Instruct
meta-llama/Meta-Llama-3-70B
meta-llama/Meta-Llama-3-70B-Instruct
```

Reply:
Thought only instruct model is cappable of chatting. I will remove it.

Reply:
> Have you tested all four of them?
> 
> ```
> meta-llama/Meta-Llama-3-8B
> meta-llama/Meta-Llama-3-8B-Instruct
> meta-llama/Meta-Llama-3-70B
> meta-llama/Meta-Llama-3-70B-Instruct
> ```

No, this is a chat template, and I tested `Meta-Llama-3-8B-Instruct`